src/SdkResolver/SdkResolver.csproj

```diff
@@ -0,0 +1,38 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(CoreSdkTargetFramework)</TargetFramework>
+    <CopyBuildOutputToOutputDirectory>false</CopyBuildOutputToOutputDirectory>
+    <GenerateDependencyFile>false</GenerateDependencyFile>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <GenerateRuntimeCofigurationFiles>false</GenerateRuntimeCofigurationFiles>
+    <AssemblyName>UNUSED</AssemblyName>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.MSBuildSdkResolver" Version="$(MicrosoftDotNetMSBuildSdkResolverPackageVersion)" ExcludeAssets="all" GeneratePathProperty="true"/>
+    <PackageReference Include="runtime.win-x86.Microsoft.NETCore.DotNetHostResolver" Version="$(MicrosoftNETCoreDotNetHostResolverPackageVersion)" ExcludeAssets="all" GeneratePathProperty="true" />
+    <PackageReference Include="runtime.win-x64.Microsoft.NETCore.DotNetHostResolver" Version="$(MicrosoftNETCoreDotNetHostResolverPackageVersion)" ExcludeAssets="all" GeneratePathProperty="true" />
+  </ItemGroup>
+
+  <Target
+    Name="GenerateLayout" 
+    Condition=" '$(OS)' == 'Windows_NT' And '$(Architecture)' == 'x86' "
+    AfterTargets="Build"
+    >
+    <Message Importance="High" Text="$(MSBuildProjectName) -&gt; $(OutputPath)" />
```

Comment:
Any reason for generating this message?

Reply:
To show some progress at minimal verbosity. This is similar to the standard, lone message from copying compiled dll to output directory but we don't do that so it doesn't show.